### PR TITLE
refactor: deployment scripts and commands

### PR DESCRIPTION
**Pull request by huaweigu**
## Summary
Improve the quality of `scripts/`. 

### Changeset
* add linting for scripts/
* add deployment and verify command for legacy `DeployTokenCallbackPlugin`

### Checklist
- [x] Did you add new tests and confirm all tests pass? (`yarn test`)
- [ ] Did you update relevant docs? (docs are found in the `docs` folder)
- [x] Do your commits follow the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/) standard?
- [x] Does your PR title also follow the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/) standard?
- [ ] If you have a breaking change, is it [correctly reflected in your commit message](https://www.conventionalcommits.org/en/v1.0.0/#examples)? (e.g. `feat!: breaking change`)
- [x] Did you run lint (`yarn lint`) and fix any issues?
- [x] Did you run formatter (`yarn format:check`) and fix any issues (`yarn format:write`)?

## Testing
No impact.

## Documentation
n/a

### CODE REVIEW

File: script/verify/DefaultTokenCallbackPlugin.json [hunk 1]
```diff
@@ -0,0 +1 @@
...lin-contracts-upgradeable/contracts/","ds-test/=lib/forge-std/lib/ds-test/src/","eth-gas-reporter/=node_modules/eth-gas-reporter/","forge-std/=lib/forge-std/","hardhat/=node_modules/hardhat/","@modular-account/=lib/modular-account/src/","@solady/=lib/solady/src/","@fcl/=lib/FreshCryptoLib/solidity/src/","@p256-verifier/=lib/p256-verifier/src/","@webauthn-sol/=lib/webauthn-sol/src/","@modular-account-libs/=lib/modular-account-libs/src/","FreshCryptoLib/=lib/FreshCryptoLib/solidity/src/","account-abstraction/=lib/account-abstraction/contracts/","erc4626-tests/=lib/openzeppelin-contracts-upgradeable/lib/erc4626-tests/","hardhat-deploy/=node_modules/hardhat-deploy/","modular-account-libs/=lib/modular-account-libs/src/","openzeppelin-contracts-upgradeable/=lib/openzeppelin-contracts-upgradeable/","openzeppelin-contracts/=lib/openzeppelin-contracts/","p256-verifier/=lib/p256-verifier/","solady/=lib/solady/src/"],"optimizer":{"enabled":true,"runs":200},"metadata":{"useLiteralContent":false,"bytecodeHash":"ipfs","appendCBOR":true},"outputSelection":{"*":{"*":["abi","evm.bytecode","evm.deployedBytecode","evm.methodIdentifiers","metadata"]}},"evmVersion":"paris","viaIR":true,"libraries":{}}}
```
Comment (dlim-circle) [inline script/verify/DefaultTokenCallbackPlugin.json:1]: Should we remove [80002_run-1725650624_DefaultTokenCallbackPlugin.json]

File: script/cmd/DeployTokenCallbackPlugin [hunk 2]
```diff
@@ -0,0 +1 @@
...001600160401b038311610125576020808501948460051b01011161012557565b90815180825260208080930193019160005b828110610b42575050505090565b83516001600160e01b03191685529381019392810192600101610b34565b80516005811015610b86576040918291845260ff60208201511660208501520151910152565b634e487b7160e01b600052602160045260246000fd5b90815180825260208080930193019160005b828110610bbc575050505090565b90919293826080600192610be483895163ffffffff60e01b8151168452015184830190610b60565b01950193929101610bae565b6040516384b9b37960e01b81526000356001600160e01b031916600482015260ff9091166024820152604490fd5b60405190608082018281106001600160401b03821117610a9e57604052606080838181528160208201528160408201520152565b6040519061018082018281106001600160401b03821117610a9e5760405281610160606091828152826020820152826040820152828082015260006080820152600060a08201528260c08201528260e08201528261010082015282610120820152826101408201520152565b805115610ccb5760200190565b634e487b7160e01b600052603260045260246000fd5b805160011015610ccb5760400190565b805160021015610ccb5760600190565b805160031015610ccb576080019056fea26469706673582212205e81eb8270352b3e36936caf6244b9804c7d4651c344dd58c7114f0f93f5902b64736f6c63430008180033 --etherscan-api-key $ETHERSCAN_API_KEY
```
Comment (dlim-circle) [inline script/cmd/DeployTokenCallbackPlugin:1]: Should we remove [80002_run-1725650624_DeployTokenCallbackPlugin]

Comment (huaweigu) [reply script/cmd/DeployTokenCallbackPlugin:1]: [done](https://github.com/circlefin/buidl-wallet-contracts/pull/12/commits/504fecab677609369782a013eb57d97bf54f6b85)

Comment (dlim-circle) [reply script/cmd/DeployTokenCallbackPlugin:1]: You removed the verify folder one but not the cmd folder

Comment (huaweigu) [reply script/cmd/DeployTokenCallbackPlugin:1]: ah okay didn't realize we have two

File: script/verify/DefaultTokenCallbackPlugin.json (hunk 1 above)
Comment (huaweigu) [reply script/verify/DefaultTokenCallbackPlugin.json]: [done](https://github.com/circlefin/buidl-wallet-contracts/pull/12/commits/ae996afe1e769a22ee325e7383d604c1ee7d1e34)

**Review by dlim-circle — APPROVED** [review]

**Pull request merged**